src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,178 @@
+package storage;
+
+import task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String directory;
+    private String fileName;
+
+    public Storage(String directory, String fileName) {
+        this.directory = directory;
+        this.fileName = fileName;
+    }
+    /**
```

Comment:
Should there be an extra line before this line?

Reply:
Thank you for pointing it out!